### DISCUSSION TITLE
Am I overqualified?

### DISCUSSION THREAD

**Discussion by LiJunYi2** [vercel/community#6643] [Help] [answered]
### Summary

I received an alert with the following

```xml
Your site is growing!Your free team xxxx-projects has used 100%of the included free tier usage for Function Invocation (100,000 Invocations).You can upgrade to Pro for more included usage and the ability to pay on-demand.
```
but，i see Usage：
<img width="925" alt="image" src="https://github.com/vercel/community/assets/40384503/28f357d4-f6b9-45ce-be77-3db2b404dec2">
<img width="911" alt="image" src="https://github.com/vercel/community/assets/40384503/4b109c43-3674-4745-a7d9-a32a7c0ae566">

I have not found Function to Invocation this usage

### Steps to Reproduce

<img width="925" alt="image" src="https://github.com/vercel/community/assets/40384503/28f357d4-f6b9-45ce-be77-3db2b404dec2">
<img width="911" alt="image" src="https://github.com/vercel/community/assets/40384503/4b109c43-3674-4745-a7d9-a32a7c0ae566">

**Comment by Huibq** [comment]
I also received this warning today. I currently suspect that this alarm is because the project has called Serverless Functions more than 100,000 times. You can check it out at this link:  https://vercel.com/docs/pricing/serverless-functions#plan-usage

**Comment by amyegan** [comment, marked as answer]
@LiJunYi2 @Huibq We're rolling out [updated pricing](https://vercel.com/blog/improved-infrastructure-pricing) in the coming months, but it looks like these notifications went out too early. The new, more granular metrics now show up on your usage page so you can [optimize](https://vercel.com/docs/pricing/serverless-functions#managing-function-invocations) before the new limits take effect. 

Free projects will not be paused for the next 6 mo if you go over the free limits.

https://vercel.com/docs/pricing/manage-and-optimize-usage

**Reply by LiJunYi2** [reply]
hello , Who should I contact if I just want to increase the usage of Serverless Functions? Is it possible to add this separately?
